setup.py

```diff
@@ -24,7 +25,8 @@
         "sidecar",
         "imageio",
         "matplotlib",
-        "scikit-learn"
+        "scikit-learn",
+        "imgui-bundle",
     ],
     "notebook": [
```

Comment:
I think we want notebook to always use imgui by default

Reply:
we want it to be as modular as possible, someone might want nb but not imgui. So if someone wants both they do `pip install fastplotlib[notebook,imgui]`